bin/yaml/asmx.yaml

```diff
@@ -0,0 +1,15 @@
+compilers:
+  asmx:
+    type: tarballs
+    compression: gz
+    targets:
+      - name: 2.0
+      url: https://github.com/langprogramming-AsmX/AsmX/releases/download/v2.0/AsmX-v2.0.tar.gz
+
+    arm:
+      type: tarballs
+      compression: gz
+      check_exe: asmx-cli -v
+      targets:
+      - name: 3.0
+      url: https://github.com/langprogramming-AsmX/AsmX/releases/download/v3.0.0/AsmX-v3.0.0.tar.gz
```

Comment:
please ensure the file ends on a new line, whatever you do though

Reply:
I'm sorry, I forgot about the empty line, I fixed it.